compiler/rustc_symbol_mangling/src/typeid/typeid_itanium_cxx_abi.rs

```diff
@@ -632,7 +632,7 @@ fn encode_ty<'tcx>(
         }
 
         // Trait types
-        ty::Dynamic(predicates, region) => {
+        ty::Dynamic(predicates, region, _repr) => {
```

Comment:
We need to mangle dyn and dyn* separately, if you don't want to fix then can you leave a FIXME?